packages/edit-site/src/components/sidebar-navigation-screen-page/page-details.js

```diff
@@ -135,12 +135,16 @@ export default function PageDetails( { id } ) {
 				  )?.title?.rendered
 				: null;
 
+			const { getEntityRecord } = select( coreStore );
+			const siteSettings = getEntityRecord( 'root', 'site' );
+
 			return {
 				parentTitle: _parentTitle,
 				templateTitle: _templateTitle,
+				postsPage: record?.id === siteSettings?.page_for_posts,
```

Comment:
Nit: I think it's better to rename the variable with something like `isPostsPage` or `isPageForPosts` to indicate the boolean type.

Also why not use the `id` directly?

Reply:
Thanks for the review Nik! 

I went with the `record.id` as `id` is of type string. I could use the `id` received as a parameter, but then I'd have to cast it or make a weaker comparison. I thought it was cleaner to use the one from the record.

Reply:
I usually cast it to number, but that's a nit. Thanks again!